planscore/website/templates/plan.html

```diff
@@ -180,6 +180,15 @@ <h4>Sensitivity Testing</h4>
 	        mm_metric_url = 'https://planscore.org/metrics/meanmedian/',
 	        model_url_pattern = '{{ url_for("get_model_description", prefix="data/2020") }}';
 
+        // preload the geometry as it can take a while to load
+        const probably_geometry_url = plan_url.replace('index.json', 'geometry.json');
```

Comment:
@migurski i'll want your feedback on this bit. :) 

in the cases i looked at, the url construction was this easy. But I see the `geom_prefix` there, so I suspect there's a more robust way of making this URL?